### PR TITLE
`connectors.md` gains Github labels for connector issues

### PR DESCRIPTION
**Pull request by evantahler**

### CODE REVIEW

File: airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java [hunk 1]
```diff
@@ -28,9 +28,10 @@ public class ConnectorMarkdownGenerator {
       .desc("name for the generated catalog json file").build();
   private static final Options OPTIONS = new Options().addOption(PROJECT_ROOT_OPTION).addOption(SEED_ROOT_OPTION).addOption(OUTPUT_FILENAME_OPTION);
 
-  private static final String githubCodeBase = "https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors";
+  private static final String githubOrgProject = "airbytehq/airbyte";
```
Comment (bnchrch) [inline airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java:31]: Good golly gee I do love constants

File: airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java [hunk 2]
```diff
@@ -102,11 +104,17 @@ private String buildMarkdownTable(final List<JsonNode> definitions, final String
       final String dockerImage = definition.get("dockerRepository").asText() + ":" + definition.get("dockerImageTag").asText();
       final String releaseStage = definition.get("releaseStage") != null ? definition.get("releaseStage").asText() : "unknown";
       final String documentationUrl = definition.get("documentationUrl") != null ? definition.get("documentationUrl").asText() : "";
-      final String docLink = !documentationUrl.equals("") ? "[link](" + documentationUrl + ")" : "missing";
-      final String codeLink = "[code](" + githubCodeBase + "/" + codeName + ")";
+      final String docLink = !documentationUrl.equals("") ? "[docs](" + documentationUrl + ")" : "missing";
+      final String issuesLabel = "connectors/" + (codeName.contains("source-") ? "source" : "destination") + "/"
```
Comment (bnchrch) [inline airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java:108]: On a scale of 1-10 on the hurt my brain to read scale this felt about a 4.

Any chance we can make that a named function or add a comment?

**Review by bnchrch — APPROVED** [review]
Approved! With one nit.

File: airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java (hunk 2 above)
Comment (evantahler) [reply airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java:108]: well... what I really hope is that this whole file goes away and #metadata makes this file soon...

Comment (evantahler) [reply airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java:109]: comment commented!

**Pull request merged**